### ISSUE TITLE
HMR doesn't work / ERR_UNSUPPORTED_ESM_URL_SCHEME

### ISSUE THREAD

**Issue by kolodziejczak-sz** [preactjs/wmr#335]
Hey guys,
I tried to test WMR on Windows. On that environment HMR works well, but I can't load the wmr.config.mjs at all. It gives me a following error:
```
Failed to load wmr.config.mjs
Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only file and data URLs are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. 
Received protocol 'c:'
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: C:\Users\foo\projects\wmr-preact\wmr.config.mjs
```
I tried somehow bypass that error and I decided to run the project with a dev container (docker). Dev container does the job with wmr.config.js but right now the HMR doesn't work whether I use my custom config or not. The socket connection is not receiving any message after file change (_wmr.js@handleMessage). My experience is similar to that comment: https://github.com/preactjs/wmr/issues/246#issuecomment-746433150

my node version is v15.8.0

**Comment by kolodziejczak-sz** [comment]
I decided to do some investigation on my own: adding the ...{ usePolling: true } to chokkidar watcher options solves the problem

**Comment by developit** [comment]
@kolodziejczak-sz The next release of WMR will fix your first Windows issue, since #347 has now been merged.

Regarding Docker - what OS are you using in the Docker container? We could probably include a list of OSes to apply the `usePolling` workaround to [here](https://github.com/preactjs/wmr/blob/a4993043f234d6ed57ca2f95cf9844c8fda01c31/packages/wmr/src/wmr-middleware.js#L116).

**Comment by JoviDeCroock** [comment]
Closing as this should be fixed now
